desc/compute/_profiles.py

```diff
@@ -133,6 +133,174 @@ def _gradpsi_mag(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="Te",
+    label="T_e",
+    units="eV",
+    units_long="electron-Volts",
+    description="Electron temperature",
+    dim=1,
+    params=["Te_l"],
+    transforms={},
+    profiles=["electron_temperature"],
+    coordinates="r",
+    data=["0"],
+)
+def _Te(params, transforms, profiles, data, **kwargs):
+    if profiles["electron_temperature"] is not None:
+        data["Te"] = profiles["electron_temperature"].compute(params["Te_l"], dr=0)
+    else:
+        data["Te"] = jnp.nan * data["0"]
+    return data
+
+
+@register_compute_fun(
+    name="Te_r",
+    label="\\partial_{\\rho} T_e",
+    units="eV",
+    units_long="electron-Volts",
+    description="Electron temperature, first radial derivative",
+    dim=1,
+    params=["Te_l"],
+    transforms={},
+    profiles=["electron_temperature"],
+    coordinates="r",
+    data=["0"],
+)
+def _Te_r(params, transforms, profiles, data, **kwargs):
+    if profiles["electron_temperature"] is not None:
+        data["Te_r"] = profiles["electron_temperature"].compute(params["Te_l"], dr=1)
+    else:
+        data["Te_r"] = jnp.nan * data["0"]
+    return data
+
+
+@register_compute_fun(
+    name="ne",
+    label="n_e",
+    units="m^{-3}",
+    units_long="1 / cubic meters",
+    description="Electron density",
+    dim=1,
+    params=["ne_l"],
+    transforms={},
+    profiles=["electron_density"],
+    coordinates="r",
+    data=["0"],
+)
+def _ne(params, transforms, profiles, data, **kwargs):
+    if profiles["electron_density"] is not None:
+        data["ne"] = profiles["electron_density"].compute(params["ne_l"], dr=0)
+    else:
+        data["ne"] = jnp.nan * data["0"]
+    return data
+
+
+@register_compute_fun(
+    name="ne_r",
+    label="\\partial_{\\rho} n_e",
+    units="m^{-3}",
+    units_long="1 / cubic meters",
+    description="Electron density, first radial derivative",
+    dim=1,
+    params=["ne_l"],
+    transforms={},
+    profiles=["electron_density"],
+    coordinates="r",
+    data=["0"],
+)
+def _ne_r(params, transforms, profiles, data, **kwargs):
+    if profiles["electron_density"] is not None:
+        data["ne_r"] = profiles["electron_density"].compute(params["ne_l"], dr=1)
+    else:
+        data["ne_r"] = jnp.nan * data["0"]
+    return data
+
+
+@register_compute_fun(
+    name="Ti",
+    label="T_i",
+    units="eV",
+    units_long="electron-Volts",
+    description="Ion temperature",
+    dim=1,
+    params=["Te_l"],
```

Comment:
params here does not match params in the compute function, should be Ti_l

Reply:
good catch